django/middleware/common.py

```diff
@@ -161,8 +161,14 @@ def is_ignorable_request(self, request, uri, domain, referer):
         """
         Returns True if the given request *shouldn't* notify the site managers.
         """
-        # '?' in referer is identified as search engine source
+        # Ignore three specific behaviours:
```

Comment:
I guess these comments could be moved to the docstring at this point. 
behaviours -> situations